### DISCUSSION TITLE
alpha or beta

### DISCUSSION THREAD

**Discussion by elite0421** [community/community#55643] [Programming Help] [answered]
### Body

I need to put the version number for my product as I made it basically. I am trying to release with basic one and get the opinions from others. I guess there are alpha and beta versions available and what I can choose in my case?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Auiux** [comment, marked as answer]
In web development, the terms "alpha" and "beta" refer to stages of software development before the final release of a product. Here's a detailed explanation of the difference between the two:

Alpha versions are released first and are typically tested by a limited audience, while beta versions are released after alpha and are typically tested by a wider audience.
Alpha versions are usually not feature-complete, while beta versions are feature-complete.
The purpose of the alpha release is to identify major issues and bugs, while the purpose of the beta release is to identify remaining issues and bugs.

I guess you need to set alpha version at first.
